{"name": "test_mine[case2]", "status": "broken", "statusDetails": {"message": "uiautomator2.exceptions.UiObjectNotFoundError: -32002 Client error: <> data: Selector [text='跳过'], method: None", "trace": "self = <test_mine.TestMine object at 0x0000020F507D7508>\ncase = {'id': 'mine_003', '优先级': 'P1', '前置条件': '', '是否执行': 'yes', ...}\n\n    @pytest.mark.parametrize(\"case\", run_ed)\n    def test_mine(self, case):\n        # case_id = case[ExcelConfig.case_id]\n        # print(case_id)\n        # case_catalog = case[ExcelConfig.case_catalog]\n        # case_name = case[ExcelConfig.case_name]\n        # case_priority = case[ExcelConfig.case_priority]\n        # case_is_run = case[ExcelConfig.case_is_run]\n        # case_preconditions = case[ExcelConfig.case_preconditions]\n        # case_step = case[ExcelConfig.case_step]\n        case_step_node = case[ExcelConfig.case_step_node]\n        date1 = analysis(case_step_node)\n        # print(date1)\n        # case_expected_outcome = case[ExcelConfig.case_expected_outcome]\n        case_expected_node = case[ExcelConfig.case_expected_node]\n        data2 = analysis(case_expected_node)\n        # print(data2)\n>       d = ExecuteU2(\"851QFDSG22CDZ\", \"com.tencent.hobby\")\n\ntestcase\\test_mine.py:33: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\nutils\\uiauto_util.py:27: in __init__\n    self.dv(text=\"跳过\").click()\nD:\\Program Files\\Python37\\lib\\site-packages\\uiautomator2\\_selector.py:150: in click\n    self.must_wait(timeout=timeout)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <uiautomator2._selector.UiObject object at 0x0000020F50848F08>\nexists = True, timeout = None\n\n    def must_wait(self, exists=True, timeout=None):\n        \"\"\" wait and if not found raise UiObjectNotFoundError \"\"\"\n        if not self.wait(exists, timeout):\n>           raise UiObjectNotFoundError({'code': -32002, 'data': str(self.selector), 'method': 'wait'})\nE           uiautomator2.exceptions.UiObjectNotFoundError: -32002 Client error: <> data: Selector [text='跳过'], method: None\n\nD:\\Program Files\\Python37\\lib\\site-packages\\uiautomator2\\_selector.py:345: UiObjectNotFoundError"}, "parameters": [{"name": "case", "value": "{'id': 'mine_003', '用例目录': '头像', '用例名称': '我的画面点击头像', '优先级': 'P1', '是否执行': 'yes', '前置条件': '', '步骤': '1.嚯app启动》点击底部【我的】tab，我的画面点击头像\\n2.再次点击头像或下拉头像', '步骤节点': '我的, com.tencent.hobby:id/user_head_icon\\n(0.424,0.702)', '预期结果': '1.头像放大表示\\n2.关闭放大头像画面，回到我的画面', '预期节点': '//android.widget.FrameLayout[2]\\ncom.tencent.hobby:id/user_head_icon'}"}], "start": 1626745223674, "stop": 1626745267834, "uuid": "952fb763-e0f2-4703-962d-62220ef4bb7f", "historyId": "22380043696c470ba798bdf9dedef50e", "testCaseId": "da5ae44c99fde797f954ed7865065f3e", "fullName": "testcase.test_mine.TestMine#test_mine", "labels": [{"name": "parentSuite", "value": "testcase"}, {"name": "suite", "value": "test_mine"}, {"name": "subSuite", "value": "TestMine"}, {"name": "host", "value": "v_linkaica-PC0"}, {"name": "thread", "value": "19620-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcase.test_mine"}]}